packages/@okta/vuepress-site/docs/guides/submit-app/overview/index.md

```diff
@@ -1,22 +1,34 @@
 
-For details on how to build a SCIM provisioning integration, see our [SCIM Provisioning Guide](/docs/guides/build-provisioning-integration/).
+## Protocols supported
 
-For details on how to build a SAML Single Sign-On integration, see our [SAML Guide](/docs/guides/build-sso-integration/).
+This guide covers app submissions that use one or more of the protocols:
 
-Our goal is to help you identity-enable your application using industry standards. The guidance offered here for SAML and SCIM allows you to integrate with customers using other identity solutions.
+* [OpenID Connect (OIDC)](https://openid.net/connect/)
+* [Security Assertion Markup Language (SAML)](https://en.wikipedia.org/wiki/SAML_2.0), version 2.0
+* [System for Cross-domain Identity Management (SCIM)](http://www.simplecloud.info/), versions 2.0 and 1.1
 
-To get your app published in the OIN:
+## Prerequisites
 
-1. Sign up for an [OIN Developer account](https://developer.okta.com/signup/).
-1. Prepare and test your app before submission.
-1. Submit your app to Okta through the OIN Manager tool.
-1. Work with Okta App Analysts who will test your application with your input and publish your app to the OIN App Catalog.
+As a prerequisite to submitting your app, you must have a functional app created and tested in accordance with one of our OIN guides:
 
-If you need additional help with the publication process, see the [Frequently Asked Questions](../faqs/), post a question on the [Okta Developer Forum](https://devforum.okta.com/) or email us at <developers@okta.com>.
+* [Build a SCIM provisioning integration](/docs/guides/build-provisioning-integration/)
+* [Build a Single-Sign On integration](/docs/guides/build-sso-integration/before-you-begin/).
+
+## Submission process
+
+After you have built a functioning app integration, a few steps are required to submit it for Okta review and publication in the OIN. In general:
+
+1. Create a customer-facing configuration guide.
+1. Submit your app to Okta through the OIN Manager tool. Your submission must provide Okta with the general and protocol-specific metadata needed to create a customized app for publication in the OIN.
+1. Work with Okta App Analysts who will test your application with your input and get your app published to the OIN App Catalog.
+
+If you need additional help with the publication process, you can post a question on the [Okta Developer Forum](https://devforum.okta.com/) or email us at <developers@okta.com>.
+
+>**Note:** OIN apps are public. If you want to submit a private app request for SCIM 1.1 apps, or Profile Master apps, or apps that use a custom header expression for Header Auth, then use the SCIM App Wizard to create your app and then send an email to <developers@okta.com>.
```

Comment:
"for Header Auth, then use" > "for Header Auth, use the SCIM" (to avoid repeating 'then' later).

Reply:
I left the "then" after the "Header Auth" and removed the second instance.